src/__stories__/text-field-story.tsx

```diff
@@ -64,22 +64,20 @@ export const Controlled: StoryComponent<TextFieldControlledArgs> = ({
         <ResponsiveLayout isInverse={inverse} fullWidth>
             <Box padding={16}>
                 <Stack space={16}>
-                    <div data-testid="text-field-wrapper">
-                        <TextField
-                            value={rawValue}
-                            onChangeValue={(value, rawValue) => {
-                                setValue(value);
-                                setRawValue(rawValue);
-                            }}
-                            name="text"
-                            autoComplete="off"
-                            data-testid="text-field"
-                            maxLength={maxLength ? 200 : undefined}
-                            endIcon={icon ? <IconMusicRegular /> : undefined}
-                            getSuggestions={suggestions ? getCountrySuggestions : undefined}
-                            {...rest}
-                        />
-                    </div>
+                    <TextField
+                        value={rawValue}
+                        onChangeValue={(value, rawValue) => {
+                            setValue(value);
+                            setRawValue(rawValue);
+                        }}
+                        name="text"
+                        autoComplete="off"
+                        dataAttributes={{testid: 'text-field'}}
+                        maxLength={maxLength ? 200 : undefined}
```

Comment:
should we enable maxLenght helper text for text fields without multiline? 

what do you think @aweell 

![image](https://github.com/Telefonica/mistica-web/assets/6722153/22935232-e084-4f28-968b-402ec6782026)

Reply:
I remember we had this conversation a few months ago and decided not to do that.

I believe that is a separate task and we should evaluate the side effects carefully before adding it.

https://teams.microsoft.com/l/message/19:380a30ed56a946c8bad31847048d0c7e@thread.tacv2/1699889469262?groupId=e265fe99-929f-45d1-8154-699649674a40&parentMessageId=1699889469262&tenantId=9744600e-3e04-492e-baa1-25ec245c6f10